pairing/circuits/src/compiled.rs

```diff
@@ -17,10 +17,10 @@ pub struct Compiled<E: PairingEngine>
     pub constraints: CsMat<E::Fr>,
 
     // compiled polynomial commitments
-    pub col_comm: E::G1Affine,
-    pub row_comm: E::G1Affine,
-    pub val_comm: E::G1Affine,
-    pub rc_comm: E::G1Affine,
+    pub col_comm: PolyComm<E::G1Affine>,
```

Comment:
Can you remove the commits which do this on the pairing version so we can make sure they work one at a time?

Reply:
This is completed: state of the pairing-based Marlin is returned to that committed in the Master branch at the point and off which this branch has been forked.